hcl/hcl.go

```diff
@@ -719,34 +727,51 @@ func parseGenerateHCLBlock(block *ast.Block) (GenHCLBlock, error) {
 	var (
 		lets    hclsyntax.Blocks
 		content *hclsyntax.Block
+		asserts []AssertConfig
 	)
 
 	err := validateGenerateHCLBlock(block)
 	if err != nil {
 		return GenHCLBlock{}, err
 	}
 
-	for _, b := range block.Body.Blocks {
-		switch b.Type {
+	errs := errors.L()
+
+	for _, subBlock := range block.Blocks {
+		switch subBlock.Type {
 		case "lets":
-			lets = append(lets, b)
+			lets = append(lets, subBlock.Block)
+		case "assert":
+			// TODO(KATCIPIS): error handling tests
```

Comment:
if we miss tests, isn't it better to be a TODO in the hcl_test.go instead?

Reply:
Ah this TODO is a leftover, sorry and thanks for the catch :+1: